src/features/auth/components/AuthForm/AuthForm.css

```diff
@@ -50,26 +50,38 @@
 }
 
 .auth-form__button {
```

Comment:
i suggest to create `Button` component, since we are going to use button from our package in the future